app/controllers/users.ts

```diff
@@ -12,16 +28,39 @@ export function getUsers(req: Request, res: Response, next: NextFunction): void
     .catch(next);
 }
 
-export function createUser(req: Request, res: Response, next: NextFunction): void {
+export async function createUser(req: Request, res: Response, next: NextFunction): Promise<void> {
+  const data = req.body;
+  const errors: string[] = [];
+  const errorEmail = await checkEmail(data.email);
+  const errorPassword = checkPassword(data.password);
+  if (errorEmail) {
+    errors.push(errorEmail);
+  }
+  if (errorPassword) {
+    errors.push(errorPassword);
+  }
+  if (errors.length) {
+    res.status(HttpStatus.FORBIDDEN).json(errors);
```

Comment:
here you should use `next()` function to pass the error to the errorHandler middleware. And you should throw a custom error, check the `app/errors.js` file